cypress/e2e/common/cmsSteps.js

```diff
@@ -416,4 +416,102 @@ And('user remembers the source of selected promo image for the mini landing', ()
     cy.get('details img').then($el => {
         imageSrc = $el[0].getAttribute('src')
     })
+});
+
+Then('user selects {string} option from Operations dropdown for content with title {string}', (translateOption, title) => {
+    cy.get(`td:contains('${title}')`).siblings('td').find(`ul.dropbutton >li> a:contains("${translateOption}")`).click({ force: true });
+});
+
+Then('the page title is {string}', (title) => {
+    cy.get(`h2:contains('${title}')`).should('be.visible');
```

Comment:
this step is one of the most common steps. -its already implemented in common/index.js - remove it here

Reply:
Here we have 'h2'. In common-> index.js we have used 'h1'
![image](https://user-images.githubusercontent.com/28455973/224403857-e7afb0c2-b4d3-4400-9339-f1cf4921fe72.png)